.github/workflows/build-publish.yaml

```diff
@@ -72,33 +56,33 @@ jobs:
           docker build --file Dockerfile -t $GITHUB_REPOSITORY:$IMAGE_TAG .
 
       - name: Save image as artifact
-        env:
-          IMAGE_TAG: ${{ needs.prepare.outputs.FULL_IMAGE_TAG }}
         run: |
-          docker save -o /tmp/container.tar.gz $GITHUB_REPOSITORY:$IMAGE_TAG
+          docker save -o /tmp/container-$IMAGE_TAG.tar.gz $GITHUB_REPOSITORY:$IMAGE_TAG
 
       - uses: actions/upload-artifact@v3
         with:
           name: container-image
-          path: /tmp/container.tar.gz
-          retention-days: 1  # Instead of default 90 days
+          path: /tmp/container-$IMAGE_TAG.tar.gz
+          retention-days: 7 # Instead of default 90 days
```

Comment:
Allued to this in the PR description, but my thinking here was:
- author submits PR, image gets built
- PR is reviewed and approved 25 hours later

I think the built image would no longer be available and the push job would fail.

This is just a larger "magic number" further in the future, but hopefully it gives us more time for the review + merge cycle.